src/processes/domain-process.js

```diff
@@ -9,7 +10,6 @@ const {
 const constant = require('../utils/constant');
 
 const postToGetstream = async (activity) => {
-  const {v4: uuidv4} = require('uuid');
   try {
     const activityId = uuidv4();
     const randomizer = randomBetweenPositiveAndNegative(constant.DAY_IN_SECONDS);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `uuidv4` function is now being called directly in the `postToGetstream` function without requiring the 'uuid' module again. This is a good change as it reduces redundancy and makes the code cleaner.
```diff
-  const {v4: uuidv4} = require('uuid');
+  const activityId = uuidv4();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->